README.md

```diff
@@ -8,10 +8,26 @@ CLI tool to compute the [TypeRefHash (TRH)](https://www.gdatasoftware.com/blog/2
 
 ## Installation
 
-Just download the binary from the [GitHub Releases](https://github.com/GDATASoftwareAG/TypeRefHasher/releases) tab. There are two versions:
+There are multiple options to obtain the *TypeRefHasher*.
 
-1. `trh` -> Linux x64
-2. `trh.exe` -> Windows x64
+### Windows
+
+Install the tool with [winget](https://github.com/microsoft/winget-cli). The tool gets added to your `PATH` environment variable.
+
+```powershell
+winget install GDATA.TypeRefHasher
```

Comment:
nit: I suppose this can only be tested once this PR is merged?

Reply:
Yeah, that's the case. I would wait the merge until the tool is on _winget_ such that we can try it at least once. Seems that this takes a lot of time as the PR is already open for a few days and still not merged by MS.